pkg/statuscodes/statuscodes.go

```diff
@@ -1,16 +1,29 @@
 // Copyright 2022 Outreach Corporation. All Rights Reserved.
 
-// Description: Provides custom status types and helpers
-
-// Package statuscodes provides custom status types and helpers
+// Description: See below
```

Comment:
```suggestion
// Description: Implements the statuscodes package.
```

More in the spirit of what @george-e-shaw-iv wanted.